### PR TITLE
added bhagavad-geeta-card

### LINKED ISSUE #38: Adding a Bhagavad Geeta Quotes Card
I would like to add a card which will display a random quote from the Bhagavad Geeta, kindly assign me this.

### PR DESCRIPTION
**Pull request by piyushkdas0611**
Screenshot (10)

closes #38

### CODE REVIEW

File: src/data/bhagavad_geeta_quotes.json
Comment (akanshSirohi) [inline src/data/bhagavad_geeta_quotes.json:1]: More quotes required

**Review by akanshSirohi — CHANGES_REQUESTED** [review]
Can you add more quotes...?

File: src/data/bhagavad_geeta_quotes.json
Comment (piyushkdas0611) [reply src/data/bhagavad_geeta_quotes.json:1]: Sure

Comment (piyushkdas0611) [reply src/data/bhagavad_geeta_quotes.json:1]: 50 quotes are good enough?

**Review by akanshSirohi — APPROVED** [review]

**Comment by akanshSirohi** [comment]
@piyushkdas0611 Your card is live now!

**Pull request merged**